src/actions/resize/ResizeActions/shared/ResizeAdvancedAction.ts

```diff
@@ -1,5 +1,5 @@
 import ResizeSimpleAction from "./ResizeSimpleAction";
-import {GravityQualifier} from "../../../../values/gravity/Gravity";
+import {IAnyGravity} from "../../../../values/gravity/GravityQualifier";
```

Comment:
IAnyGravity name is a little weird why was this changed from GravityQualifier?

Reply:
Gravity is composed of four, unrelated classes: Auto, Focus, Compass and OCR.
They are all "Gravity Qualifiers", but not all places that accept Gravity, accept *all* gravity types.

IAnyGravity is a type (not a class), that attemps to signal just that - you can pass any of the four types of Gravity.
If everything would just accept a common GravityQualifier, we would granularity over what function can accept what.

Furthermore, Gravity itself has qualifiers, which makes the terminology a bit complex. this resolves that as well.